### ISSUE TITLE
Map not displayed correctly

### ISSUE THREAD

**Issue by Bartzi** [meet-here/frontend#14]
when changing from one menu entry to another and back to the map display, the map is not displayed correctly: 

![image](https://cloud.githubusercontent.com/assets/2399349/8606315/1473378a-268b-11e5-96b5-b6edb89c3a90.png)

Investigate and fix!

**Comment by sknebel** [comment]
Leaflet has to be notified when it's size changes to recalculate stuff. Apparently it gets create in one size and then resized to full size, possibly by angular material?

Compare http://leafletjs.com/reference.html#map-invalidatesize
and see https://github.com/tombatossals/angular-leaflet-directive/issues/818

The first answer suggests calling

```
leafletData.getMap().then(function(map) {
    map.invalidateSize();
  });
```

after the resize happens (I'm not sure how we can get that event, I certainly don't want it called in a timer...), the author of the leaflet-directive suggests not recreating the directive, which also seems sensible -> a user nearly always will return to the map view and there is no point in recreating the map, reloading the tiles, applying all changes again, ...

TL;DR: we probably shouldn't recreate the map-view, but just hide it when we are in other screens.

**Comment by sknebel** [comment]
This seems now the default, also the "action button" is now positioned in the middle (Firefox 41.0.2 / Win7)

![unbenannt_](https://cloud.githubusercontent.com/assets/1082036/10863599/91ddf7b4-7fd2-11e5-8df1-ed5a2949b684.png)
